code/modules/clothing/gloves/color.dm

```diff
@@ -163,7 +163,7 @@
 	item_state = "lgloves"
 	siemens_coefficient = 0.3
 	permeability_coefficient = 0.01
-	item_color="white"
+	item_color="latex"

```

Comment:
There is no latex crayon, make it it use the mime one since you're removing white gloves from it.

Reply:
I don't think it makes any sense that washing something with a crayon (or a stamp) would make gloves translucent like latex gloves are. I can change that if you really want but I don't see how that would make sense

Reply:
you're better off closing the PR if you think that to be better than current standing @vuonojenmustaturska .

The mime crayon making white gloves at least makes sense in the fact that white gloves are an otherwise mime-exclusive (or maint drop I guess).

Reply:
Also

>There's honestly plenty of latex gloves in medbay if you needed them for a particular reason over white gloves. It's not like they're a hot item lol.

Reply:
latex item color is still not a thing

Reply:
That's the idea. What do you propose would be able to make opaque gloves translucent when washed with it?

Reply:
Mime crayon since mime stuff is invisible and your mixing visible gloves and mime invisibility to get semi-transparent gloves

Reply:
That's enough justification for me.